packages/manager/modules/pci/src/projects/new/payment/component.js

```diff
@@ -7,6 +7,8 @@ export default {
   template,
   bindings: {
     callback: '<',
+    voucherPlan: '<',
+    isDisplayablePaypalChargeBanner: '<',
```

Comment:
```suggestion
    shouldDisplayPaypalBanner: '<',
```

Reply:
Yep i have also hesitate with this name.
I preferred to add the "charge" to inform about the PayPal banner nature.

Reply:
IMHO, I find @marie-j's suggestion more appropriate in this scenario. Not a blocker but might be interesting to take in into account.